### ISSUE TITLE
task distribution: reporting + haybaler 

### ISSUE THREAD

**Issue by colindaven** [MHH-RCUG/nf_wochenende#21]
Hi @LisaHollstein 

do you think you could have a go at porting the `reporting` step, then later the `haybaler` steps, into the current nextflow scheme?

That would be great.

One problem is needing a server with R and the appropriate tools installed. We should stick to specifying one server I think, and perhaps check that all tools are installed as expected before starting ? What do you think ? It's complex, so it needs to be attempted and the problems worked through.

I will continue with `plots `and then `growth rate` and `raspir`, so we don't get in each other's way too much (I hope).

Or would you prefer another plan ?

**Comment by LisaHollstein** [comment]
Hi @colindaven 

the plan sounds good 👍

> We should stick to specifying one server I think, and perhaps check that all tools are installed as expected before starting ?

I also agree with that

**Comment by colindaven** [comment]
- [ ] plots - initial attempt completed. 
- [ ] plots - Getting bigger data for full test
- [ ] growth_rate  - prototyping started 12.7
- [ ] x
- [ ] raspir - todo
